comit/src/asset/ethereum/erc20.rs

```diff
@@ -22,11 +27,18 @@ impl Erc20Quantity {
         self.0.to_str_radix(10)
     }
 
-    pub fn from_wei_dec_str(str: &str) -> Result<Self, Error> {
+    pub fn try_from_wei_dec_str(str: &str) -> Result<Self, Error> {
         let int = BigUint::from_str_radix(str, 10)?;
         Ok(Self::try_from_wei(int)?)
     }
 
+    /// Create a erc20 quantity from a float string. E.g., this can be used to
+    /// create a DAI quantity from the ergonomic string "1234.56".
+    pub fn try_from_float(eth: &str) -> anyhow::Result<Self> {
```

Comment:
```suggestion
    pub fn try_from_eth_str(eth: &str) -> anyhow::Result<Self> {
```
Let's try to keep the function names similar.

Reply:
Well picked up (the inconsistency), thanks. Will keep in mind (the ethereum stuff is on hold until float_math goes in).